api/HTMLTrackElement.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLTrackElement",
         "support": {
           "chrome": {
-            "version_added": "23"
+            "version_added": "17"
```

Comment:
@vinyldarkscratch can you break out these fixes into a separate PR?

Reply:
These changes fix consistency, so I'm not able to remove them from this PR.

Reply:
Ah, OK then. (They could be broken out and landed first, but too much work.)

Reply:
Oh, but actually, all that remains of this PR is now updating these version numbers. But then lots of subfeatures are left as "23" and this was all introduced together, so that must be wrong.

It's not super important to fix this data now, but if you want to, can you update more of the track-related data together in this PR?